river/LockManager.zig

```diff
@@ -223,7 +226,9 @@ fn handleUnlock(listener: *wl.Listener(void)) void {
             seat.setFocusRaw(.none);
 
             // Exit locked mode
-            seat.enterMode(seat.prev_mode_id);
+            assert(seat.mode_id == 1);
+            assert(seat.mode_lifetime == .continuous);
```

Comment:
Whoops, don't merge yet, these asserts are invalid; I forgot that enter-mode works in locked mode too.

I'll remove them once I am in front of a computer again.